README.md

```diff
@@ -135,132 +135,142 @@ You can:
+                # old property: new property
+                'name': 'var'
 ```
 
 ### Change a class constant name
 
 ```yml
-rectors:
+services:
     Rector\Rector\Dynamic\ClassConstantReplacerRector:
-        # class
-        'Symfony\Component\Form\FormEvents':
-            # old constant: new constant
-            'PRE_BIND': 'PRE_SUBMIT'
-            'BIND': 'SUBMIT'
-            'POST_BIND': 'POST_SUBMIT'
+        $oldToNewConstantsByClass:
+            # class
+            'Symfony\Component\Form\FormEvents':
+                # old constant: new constant
+                'PRE_BIND': 'PRE_SUBMIT'
+                'BIND': 'SUBMIT'
+                'POST_BIND': 'POST_SUBMIT'
 ```
 
 ### Change parameters type hinting according to the parent type
 
 ```yml
-rectors:
+services:
     Rector\Rector\Dynamic\ParentTypehintedArgumentRector:
-        # class
-        'PhpParser\Parser':
-            # method
-            'parse':
-                # parameter: typehint
-                'code': 'string'
+        $typehintForArgumentByMethodAndClass:
+            # class
+            'PhpParser\Parser':
+                # method
+                'parse':
+                    # parameter: typehint
+                    'code': 'string'
 ```
 
 ### Change argument value or remove argument
 
 ```yml
-rectors:
+services:
     Rector\Rector\Dynamic\ArgumentReplacerRector:
-        -
-            class: 'Symfony\Component\DependencyInjection\ContainerBuilder'
-            method: 'compile'
-            position: 0
-            # change default value
-            type: 'changed'
-            default_value: false
+        $argumentChangesByMethodAndType:
+            -
+                class: 'Symfony\Component\DependencyInjection\ContainerBuilder'
+                method: 'compile'
+                position: 0
+                # change default value
+                type: 'changed'
+                default_value: false
 
-            # or remove
-            type: 'removed'
+                # or remove
+                type: 'removed'
 
-            # or replace old default value by new one
-            type: 'replace_default_value'
-            replace_map:
-                'Symfony\Component\DependencyInjection\ContainerBuilder::SCOPE_PROTOTYPE': false
+                # or replace old default value by new one
+                type: 'replace_default_value'
+                replace_map:
+                    'Symfony\Component\DependencyInjection\ContainerBuilder::SCOPE_PROTOTYPE': false
 ```
 
 ### Replace the underscore naming `_` with namespaces `\`
 
 ```yml
-rectors:
+services:
     Rector\Rector\Dynamic\PseudoNamespaceToNamespaceRector:
-        # old namespace prefix
-        - 'PHPUnit_'
-        # exclude classes
-        - '!PHPUnit_Framework_MockObject_MockObject'
+        $configuration:
+            # old namespace prefix
+            - 'PHPUnit_'
+            # exclude classes
+            - '!PHPUnit_Framework_MockObject_MockObject'
```

Comment:
Now it could be another param with explicit name, like `$skipClasses`
To show this feature to the user without commenting everything